### DISCUSSION TITLE
Add tick dataframe to MT5.GetSymbol() function

### DISCUSSION THREAD

**Discussion by R470R** [Kinzel/mt5R#8] [Ideas]
Hello people, first of all let's be grateful for this incredible package that we have in hands, with such useful functions and ease of use , thanks to Mr.Kinzel hard & smart work and great generosity to share.

Do you think would be useful for those AI's and statistical studies to implement tick interval in GetySymbol() function? 

Mr. Kinzel do you agree ? I would perform it myself but I'm not even near your level yet, I hope with my evolution Î could contrubute for the community too .

Do you think is very complex ? Or is just a new add in the iTF field in MT5.GetSymbol() and MT5.Quick_GetSymbol() functions, since mt5 exports tick data to ...

Best Regards thanks a lot, please leave a like if you like  👍

**Comment by R470R** [comment]
Since imported Metatrader tick data does not have volume we can close this thread as result would not be valuable.

Best Regards

**Comment by Kinzel** [comment]
Hi @R470R ! I hope you're having a good day!

Actually, I'm developing a feature pretty next from what you want. In **stock brokers** we have Time & Sales, it's a table of all deals, time and volume. Image below:

![Image 023](https://user-images.githubusercontent.com/42455433/107145539-18739f80-6921-11eb-92fc-0cc42f894a40.png)

I'm developing proper functions to fetch the table above. For Forex brokers I think there is none volume, but maybe any data below from M1 data-frame should be nice, ya? I'll check what can be done in this week.

From MT5 oficial site:

> What is Time & Sales?
> The Time & Sales feature provides the price and time of every trade executed on the exchange. Information on every trade includes the time when the trade was executed, its direction (buying or selling), as well as the price and volume of the trade. For easy visual analysis, different colors are used to indicate different trade directions: blue is used for Buy trades, pink for Sell trades, green means undefined direction. Trade volumes are additionally displayed in a histogram.

Thanks for the suggestion! 


[https://www.metatrader5.com/en/releasenotes/terminal/1357](url)
[https://www.investopedia.com/terms/t/time-and-sales.asp](url)

**Reply by R470R** [reply]
That is awesome Mr Kinzel!

Best regards
